eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -36,14 +36,28 @@ parameters:
 - name: TestTimeoutInMinutes
   type: number
   default: 60
+- name: MatrixConfigs
+  type: object
+  default:
+    - Name: NET_ci_test_base
+      Path: eng/pipelines/templates/stages/platform-matrix.json
```

Comment:
Nit: you used an absolute path in other places - changing some in this PR - shouldn't this one be too?

Reply:
I generally try to use the absolute pathing in template files that will get included/rendered by azure pipelines, as it treats `/` as the repo root for file search. This value, however, ends up getting passed to a powershell script, where I believe `/` does not correspond to the default working directory, so I use a relative path instead.